pkgs/applications/misc/openring/default.nix

```
@@ -0,0 +1,24 @@
+{ buildGoModule, fetchFromSourcehut, lib }:
+buildGoModule rec {
+  pname = "openring";
+  version = "master";
```

Should I use something different for the version? Maybe this?

```suggestion
  version = "unstable-2021-04-03";
```

---

Yes please.